exotica_core/src/problems/time_indexed_problem.cpp

```diff
@@ -38,628 +34,628 @@ REGISTER_PROBLEM_TYPE("TimeIndexedProblem", exotica::TimeIndexedProblem)
 {
-    Flags = KIN_FK | KIN_J;
+    flags_ = KIN_FK | KIN_J;
 }
 
 TimeIndexedProblem::~TimeIndexedProblem() = default;
 
-Eigen::MatrixXd TimeIndexedProblem::getBounds() const
+Eigen::MatrixXd TimeIndexedProblem::GetBounds() const
 {
-    return scene_->getKinematicTree().getJointLimits();
+    return scene_->GetKinematicTree().GetJointLimits();
 }
 
 void TimeIndexedProblem::Instantiate(TimeIndexedProblemInitializer& init)
 {
     init_ = init;
 
-    N = scene_->getKinematicTree().getNumControlledJoints();
+    N = scene_->GetKinematicTree().GetNumControlledJoints();
 
-    W_rate = init_.Wrate;
-    W = Eigen::MatrixXd::Identity(N, N) * W_rate;
-    if (init_.W.rows() > 0)
+    w_rate_ = init_.wrate;
+    W = Eigen::MatrixXd::Identity(N, N) * w_rate_;
+    if (init_.w.rows() > 0)
     {
-        if (init_.W.rows() == N)
+        if (init_.w.rows() == N)
         {
-            W.diagonal() = init_.W * W_rate;
+            W.diagonal() = init_.w * w_rate_;
         }
         else
         {
-            throw_named("W dimension mismatch! Expected " << N << ", got " << init_.W.rows());
+            ThrowNamed("W dimension mismatch! Expected " << N << ", got " << init_.w.rows());
         }
     }
 
-    if (init.LowerBound.rows() == N)
+    if (init.lower_bound.rows() == N)
     {
-        scene_->getKinematicTree().setJointLimitsLower(init.LowerBound);
+        scene_->GetKinematicTree().SetJointLimitsLower(init.lower_bound);
     }
-    else if (init.LowerBound.rows() != 0)
+    else if (init.lower_bound.rows() != 0)
     {
-        throw_named("Lower bound size incorrect! Expected " << N << " got " << init.LowerBound.rows());
+        ThrowNamed("Lower bound size incorrect! Expected " << N << " got " << init.lower_bound.rows());
     }
-    if (init.UpperBound.rows() == N)
+    if (init.upper_bound.rows() == N)
     {
-        scene_->getKinematicTree().setJointLimitsUpper(init.UpperBound);
+        scene_->GetKinematicTree().SetJointLimitsUpper(init.upper_bound);
     }
-    else if (init.UpperBound.rows() != 0)
+    else if (init.upper_bound.rows() != 0)
     {
-        throw_named("Lower bound size incorrect! Expected " << N << " got " << init.UpperBound.rows());
+        ThrowNamed("Lower bound size incorrect! Expected " << N << " got " << init.upper_bound.rows());
     }
 
-    useBounds = init.UseBounds;
+    use_bounds = init.use_bounds;
 
-    Cost.initialize(init_.Cost, shared_from_this(), CostPhi);
-    Inequality.initialize(init_.Inequality, shared_from_this(), InequalityPhi);
-    Equality.initialize(init_.Equality, shared_from_this(), EqualityPhi);
+    cost.Initialize(init_.cost, shared_from_this(), cost_phi);
+    inequality.Initialize(init_.inequality, shared_from_this(), inequality_phi);
+    equality.Initialize(init_.equality, shared_from_this(), equality_phi);
 
-    T = init_.T;
-    qdot_max = init_.JointVelocityLimit;
-    applyStartState(false);
-    reinitializeVariables();
+    T_ = init_.t;
```

Comment:
Ouch. Here our entire automatic camel casing etc. is breaking: We can't automatically do it for mathematical/physical quantities as now the XML/Python would have one meaning and the C++ another. Further, we would require checks that an automatic changed variable isn't shadowing or clashing (I did not review the initializer generation Python in full) - i.e. what if my initialiser has `delta` and `Delta` (because in the paper two different meanings are assigned to them).